src/plugins/console/public/application/containers/editor/monaco/hooks/use_register_keyboard_commands.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { useRef } from 'react';
+import { i18n } from '@kbn/i18n';
+import { monaco } from '@kbn/monaco';
+// eslint-disable-next-line @kbn/eslint/module_migration,@kbn/imports/no_unresolvable_imports
+import { IDisposable } from 'monaco-editor';
```

Comment:
nit: we should import the interface to kbn/monaco package and export it from there

Reply:
Ah I actually found an easier solution - using `monaco.IDisposable` instead of importing it separately. 😄  Added this with [ad6ab27](https://github.com/elastic/kibana/pull/183504/commits/ad6ab274403cb52c185d5b03b830fc6123ad7ea5).